### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by magalhas** [relax/relax#53]
- How to create a custom component
- Catalogue of available components

**After #51 and #42**:
- How to create containers
- How to trigger mutations
- Catalogue of available mutations and queries
- How to create new mutations and queries

**Comment by magalhas** [comment]
I think all of this should first be written in simple markdown documents into the repository wiki. Later we can think about having a nice and beautiful documentation page outside of here :)

**Comment by safoorsafdar** [comment]
yeah exactly, document should prepare first, so other can easily contribute as well with same pattern and rules.

**Comment by bruno12mota** [comment]
Shouldn't it provide a bit more than that structure? Was thinking in something like this:

* Introduction
* Structure (code structure)
* Components
  * Settings
  * Props schema
  * Classes
  * Styles
* GraphQL
  * Queries
  * Mutations
* Mongoose models
* Migration scripts
* Contributing

Any ideas?

**Comment by bruno12mota** [comment]
https://github.com/relax/relax/wiki just some drafts

**Comment by magalhas** [comment]
Lets start working on the wiki and we'll se how it goes. Overall I agree with the structure you proposed.
